extended/src/main/java/apoc/vectordb/Milvus.java

```diff
@@ -40,6 +41,21 @@ public class Milvus {
     @Context
     public URLAccessChecker urlAccessChecker;
 
+    @Procedure("apoc.vectordb.milvus.info")
+    @Description("apoc.vectordb.milvus.info(hostOrKey, collection, $configuration) - Get information about specified existing collection")
+    public Stream<MapResult> info(@Name("hostOrKey") String hostOrKey, @Name("collection") String collection, @Name(value = "dbName", defaultValue = "default") String dbName,  @Name(value = "configuration", defaultValue = "{}") Map<String, Object> configuration) throws Exception {
```

Comment:
Non so se è necessario il parametro `dbName`, 
controlla ed in caso toglilo,
altrimenti aggiungilo sulla documentazione

Reply:
dbName andrebbe inserito, l'ho preso dalla documentazione:
https://milvus.io/docs/manage-collections.md#View-Collections